client/src/components/Signup.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import { UikInput, UikButton } from "../@uik";
+import "../@uik/styles.css";
+import axios from "axios";
+import styled from "styled-components";
+
+const Container = styled.div`
+  display: flex;
+  width: 400px;
+  margin: 0 100px;
+  flex-direction: column;
+`;
+
+class Signup extends Component {
+  state = {
+    username: "",
+    password: "",
+    department: ""
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+
+    const endpoint = "http://localhost:3000/api/register";
+    axios
+      .post(endpoint, this.state)
+      .then(res => {
+        console.log(res.data);
```

Comment:
Make sure to remove all testing console.log statements before committing your work to github. It is a good habit to get into now